I use uucp over a pseudo-tty allocated by ssh with the i protocol.
Especially I call several peers via this method.

With the attached patch I can use one single port definition like 
port ssh
type pipe
command /usr/bin/ssh &lt;lots of ssh options&gt; \H
instead of one port definitioon per peer.The sys definitions then all
look like
system uucp-peer
address uucppeer.example.com
port ssh

Besides, the built-in tcp port could be replaced with this patch and
netcat, or the tty port with chat.

I thought it might be of general interest.

--- uucp-1.07/unix/pipe.c
+++ uucp-1.07/unix/pipe.c	1999-10-18 21:46:07.000000000 +0000
@@ -199,6 +199,7 @@ fspipe_dial (qconn, puuconf, qsys, zphon
   struct ssysdep_conn *q;
   int aidescs[3];
   const char **pzprog;
+  char **p;
 
   q = (struct ssysdep_conn *) qconn->psysdep;
 
@@ -211,6 +212,11 @@ fspipe_dial (qconn, puuconf, qsys, zphon
       ulog (LOG_ERROR, "No command for pipe connection");
       return FALSE;
     }
+  
+  /* Look for a string \H and replaced it by the address given for this system */
+  for (p=pzprog; *p; p++)
+    if (!strcmp(*p, "\\H"))
+      *p = zphone;
 
   aidescs[0] = SPAWN_WRITE_PIPE;
   aidescs[1] = SPAWN_READ_PIPE;
